src/cctag/utils/Exceptions.hpp

```diff
@@ -5,9 +5,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-#ifndef _CCTAG_EXCEPTIONS_HPP_
-#define _CCTAG_EXCEPTIONS_HPP_
+#pragma once
 
+#ifndef _GNU_SOURCE
```

Comment:
Need a comment with links to explain why we have that here.

Reply:
done